src/app.ts

```diff
@@ -123,6 +128,10 @@ async function app() {
     process.exit(1);
   }
 
+  // Load alias file
+  const aliasFile = getAliasFile(program.aliasFile);
+  const aliases = AliasProcessor.process(aliasFile);
+
```

Comment:
Since the alias file is optional, I think it would be useful to check to see if the option has a value before calling `getAliasFile`. With the current implementation, an info message of `Using alias file: ` will be logged when this option is not used.